flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -130,6 +130,15 @@ state.backend: jobmanager
 
 
 #==============================================================================
+# YARN
+#==============================================================================
+
+# Default number of YARN container to allocate (=Number of Task Managers)
+
+yarn.defaultNumberOfTaskManagers: 1
```

Comment:
Can you remove the configuration value from the default configuration again?

I would like to force new users to specify the number of yarn containers when they start Flink on YARN.
The configuration value is meant for production users which want to control everything using configuration values.